client/src/pages/QuestItemClicked.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useCallback, useContext, useEffect, useState } from "react";
+
+// components
+import { Leaderboard, Loading } from "@components";
+
+// utils
+import { backendAPI } from "@utils/backendAPI";
+
+// context
+import { GlobalStateContext } from "@context/GlobalContext";
+
+export const QuestItemClicked = () => {
+  const [message, setMessage] = useState("");
+  const [collectedText, setCollectedText] = useState("");
+  const [isLoading, setIsLoading] = useState(true);
+
+  // context
+  const { keyAssetImage, hasInteractiveParams } = useContext(GlobalStateContext);
+
+  useEffect(() => {
+    if (hasInteractiveParams) {
+      backendAPI.post("/quest-item-clicked")
+        .then((result) => {
+          const { addedClick, numberAllowedToCollect, totalCollectedToday, success } = result.data;
+
+          if (addedClick) {
+            setCollectedText(`${totalCollectedToday}/${numberAllowedToCollect} collected today`);
+            setMessage(`🎉 Congratulations! You are one step closer to completing your daily quest!`);
```

Comment:
I think constants like these should be in a global constants file, to make things easier to track.